ci/Jenkinsfiles/build.groovy

```diff
@@ -156,7 +156,6 @@ pipeline {
           }
           nxWithGitHubStatus(context: 'ftests') {
             echo "Functional tests are disabled (https://jira.nuxeo.com/browse/WEBUI-1260 and https://jira.nuxeo.com/browse/WEBUI-1252)"
```

Comment:
Remove the echo if ftests are being enabled

Reply:
done